### PR TITLE
Nav bar imporvement

### PR DESCRIPTION
**Pull request by Lrthayer**
added bootstrap for nav bar, changed the external css file to dashboard.css and played with the icon sizes.

### CODE REVIEW

File: app/views/login.pug [hunk 1]
```diff
@@ -10,9 +10,9 @@ html(lang="en")
         script(src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js")
 
 
-    body(style='width:100%;height:100%;background-image:url("../assets/background.png")')
+    body(style='width:100%;height:100%;background-image:url("/assets/background.png")')
 
-        img(style='display: block; margin: auto; height:250px; margin-top:50px; ' src='../assets/hive-logo2.png')
+        img(style='display: block; margin: auto; height:250px; margin-top:50px; ' src='/assets/hive.png')
```
Comment (DannyPeck) [inline app/views/login.pug]: Perfect, glad we changed the file names.

File: app/views/dashboard.pug [hunk 2]
```diff
@@ -20,19 +20,19 @@ html(lang="en")
...
 
-            div(style='position:absolute;width:100%;height:20%;left:0%;top:40%;')
-                input(id="compose" style='width:100%; height:100%;' type='image' src="/assets/sent2.png" alt='Submit')
+            li(style='position:absolute;width:100%;height:20%;left:0%;top:40%;padding-top:25px; padding-bottom:25px')
+                input(id="compose" style='width:100%; height:100%;' type='image' src="/assets/sent.png" alt='Submit' href='#')
 
-            div(style='position:absolute;width:100%;height:20%;left:0%;top:60%;')
-                input(class="contacts" style='width:100%; height:100%;' type='image' src="/assets/Contacts2.png" alt='Submit')
+            li(style='position:absolute;width:100%;height:20%;left:0%;top:60%;')
+                input(class="contacts" style='width:100%; height:100%;' type='image' width='230' height='230' src="/assets/Contacts.png" alt='Submit' href='#')
```
Comment (DannyPeck) [inline app/views/dashboard.pug]: What was the purpose of setting the width and height properties?

File: app/views/dashboard.pug [hunk 3]
```diff
@@ -20,19 +20,19 @@ html(lang="en")
                 h1 Welcome to Hive!
 
         // Side Button Bar
-        div(id='navBar' style='position:absolute;width:10%;height:100%;left:0%;top:0%;border-style:solid;background-color:rgb(49,60,80);')
+        nav(class="navbar navbar-nav" id='navBar' style='position:absolute;width:10%;height:100%;left:0%;top:0%;border-style:solid;background-color:rgb(49,60,80);')
 
-            div(style='position:absolute;width:100%;height:20%;left:0%;top:0%;')
-                input(id='home' style='width:100%; height:100%;' type='image' src="/assets/hive-logo2.png" alt='Submit')
+            li(class='active' style='position:absolute;width:100%;height:20%;left:0%;top:0%; padding-top:25px; padding-bottom:25px' )
+                input(id='home' style='width:100%; height:100%;' type='image' src="/assets/hive.png" alt='Submit' href='#')
```
Comment (DannyPeck) [inline app/views/dashboard.pug]: What does href='#' do?

File: app/views/dashboard.pug [hunk 4]
```diff
@@ -20,19 +20,19 @@ html(lang="en")
                 h1 Welcome to Hive!
 
         // Side Button Bar
-        div(id='navBar' style='position:absolute;width:10%;height:100%;left:0%;top:0%;border-style:solid;background-color:rgb(49,60,80);')
+        nav(class="navbar navbar-nav" id='navBar' style='position:absolute;width:10%;height:100%;left:0%;top:0%;border-style:solid;background-color:rgb(49,60,80);')
```
Comment (DannyPeck) [inline app/views/dashboard.pug:23]: Are these both bootstrap classes?

**Review by DannyPeck — CHANGES_REQUESTED** [review]
I noticed when testing it that the buttons have different hit boxes, can we see if we can change that so that the buttons have the same hit boxes but we shrink the image?  Can probably use padding for this.

File: app/views/dashboard.pug (hunk 4 above)
Comment (Lrthayer) [reply app/views/dashboard.pug:23]: when you say both, you mean navbar and navbar-nav?

File: app/views/dashboard.pug (hunk 3 above)
Comment (Lrthayer) [reply app/views/dashboard.pug]: it's suppose to pass the active class, it didn't really make sense to me but that is what I found via bootstrap tutorial, it didn't work anyway so I'll just get rid of those.

File: app/views/dashboard.pug (hunk 2 above)
Comment (Lrthayer) [reply app/views/dashboard.pug]: ah, forgot to change that back... it was for testing. I will get rid of that.

File: app/views/dashboard.pug (hunk 4 above)
Comment (Lrthayer) [reply app/views/dashboard.pug:23]: if so, yes. I'm pretty sure it let's bootstrap know it's not a default navbar, that way it doesn't try to force a horizontal bar.

**Review by timwhitf — APPROVED** [review]

**Pull request merged**